search/search.py

```diff
@@ -11,7 +14,29 @@ def get_search_blueprint() -> Blueprint:
 
     @blueprint.route('/search', methods=['GET'])
     def search() -> Response:
+
+        # get query string and decode bytes. If present add it to es query
+        kwargs = {}
+        query_string = request.query_string.decode('utf-8')
+        if query_string:
+            kwargs.update(dict(q=query_string))
+
+        # get all documents or query for search term
+        search = Elasticsearch(hosts=ELASTICSEARCH_HOSTS)
+        # pylint: disable=unexpected-keyword-arg
+        hits = search.search(index='_all', filter_path=['hits.hits._source'], **kwargs)
+        # pylint: enable=unexpected-keyword-arg
+        if hits:
+            # get relevant information from results
+            hits = hits.get('hits', {}).get('hits', [])
+            hits = [hit['_source'] for hit in hits if hit.get('_source')]
+
+        # create xml response
         root = etree.Element('item-list', nsmap=NAMESPACE_MAP)
```

Comment:
I assume subelements will have the `None` namespace from the map, which is `http://libero.pub`

Reply:
This sets `http://libero.pub` as the default namespace.